src/Employee.java

```diff
@@ -0,0 +1,45 @@
+import java.text.NumberFormat;
+
+public class Employee {
+    private static final NumberFormat numberFormat = NumberFormat.getCurrencyInstance();
```

Comment:
```suggestion
    private static final NumberFormat NUMBER_FORMAT = NumberFormat.getCurrencyInstance();
```
т.к. static и final одновременно